apps/upload-api/.env.sample

```
@@ -16,4 +16,6 @@ POSTGRES_PORT=5432
 POSTGRES_DB_NAME=upload
 LOGTO_M2M_PROFILE_APP_SECRET=profile_reader_local_secret
 LOGTO_M2M_PROFILE_APP_ID=ddl4llp30risjwcjymqw3
+LOGTO_M2M_SCHEDULER_APP_SECRET=scheduler_writer_local_secret
+LOGTO_M2M_SCHEDULER_APP_ID=bjd8knt92hxslvrfyezp7
 PROFILE_BACKEND_URL=http://localhost:8003
```

```suggestion
PROFILE_BACKEND_URL=http://localhost:8003
SCHEDULER_BACKEND_URL=http://localhost:8005
```

`SCHEDULER_BACKEND_URL` is missing here and in  config